tests/cypress/e2e/api/jahiaExceptions.spec.begin.ts

```diff
@@ -0,0 +1,66 @@
+import {healthCheck} from '../../support/gql';
+
+describe('Jahia exceptions probe test', () => {
+    const sshCommands = [
+        'config:list "(service.pid=org.jahia.modules.sam.healthcheck.ProbesRegistry)"'
+    ];
+
+    const waitUntilOptions = {
+        interval: 250,
+        timeout: 5000,
+        errorMsg: 'Failed to verify configuration update'
+    };
+
+    const waitUntilTestFcnDisable = (response: string) => response.indexOf('probes.JahiaExceptions.severity = IGNORED') !== -1;
+    const waitUntilTestFcnEnable = (response: string) => response.indexOf('probes.JahiaExceptions.severity = LOW') !== -1;
+
+    before('Activate the probe', () => {
+        cy.runProvisioningScript({fileName: 'jahiaExceptionsProbe/enable.json'});
```

Comment:
Why do you need to enable the probe ? Isn't it enabled by default and only executed if the severify is set to low ?

Reply:
Indeed, it's not necessary here